projects/js-packages/ai-client/src/chrome-ai/factory.ts

```diff
@@ -75,13 +75,43 @@ export default async function ChromeAIFactory( promptArg: PromptProp ) {
 		}
 	}
 
+	// Early return if the prompt type is not supported.
+	if (
+		! promptType.startsWith( 'ai-assistant-change-language' ) &&
+		! promptType.startsWith( 'ai-content-lens' )
+	) {
+		return false;
+	}
+
+	// If the languageDetector is not available, we can't use the translation or summary features—it's safer to fall back
+	// to the default AI model than to risk an unexpected error.
+	if (
+		! ( 'ai' in self ) ||
```

Comment:
This check is failing, while I get `ai` object, `languageDetector` is not there. What I could dig about it is the [LanguageDetector API](https://developer.chrome.com/docs/ai/language-detection) might have changed its implementation. Once I enable the experimental flag, I get LanguageDetector's create and availability.